### PR TITLE
Updating MI operations swagger

### PR DESCRIPTION
**Pull request by toki95**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by weidongxu-microsoft — APPROVED** [review]

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json [hunk 1]
```diff
@@ -253,6 +253,57 @@
...
+          "readOnly": true
+        },
+        "operationSteps": {
+          "$ref": "#/definitions/ManagedInstanceOperationSteps",
+          "description": "The operation steps.",
+          "readOnly": true
+        }
+      }
+    },
+    "ManagedInstanceOperationParametersPair": {
```
Comment (majastrz) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json:269]: The name of the definition will appear as a type/class name in the generated SDKs. Are these guaranteed to be pairs forever and ever?

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json [hunk 2]
```diff
@@ -253,6 +253,57 @@
           "description": "Whether the operation can be cancelled.",
           "type": "boolean",
           "readOnly": true
+        },
+        "operationParameters": {
```
Comment (majastrz) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json:257]: Are the new properties read-only as the swagger indicates?

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json (hunk 1 above)
Comment (majastrz) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json:269]: If not, you should rename the definition.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json (hunk 2 above)
Comment (toki95) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json:257]: Yes, they can not be changed.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json (hunk 1 above)
Comment (toki95) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json:269]: Yes.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Comment by weidongxu-microsoft** [comment]
@toki95 All pass. Let me know when you need it merged.

**Comment by toki95** [comment]
Please merge

**Pull request merged**